screwdriver.yaml

```diff
@@ -10,6 +11,17 @@ jobs:
       - bundle_install: bundle install
       - build: bundle exec jekyll build --source docs --destination _site
 
+  publish:
+      environment:
+          RELEASE_FILE: guide.tgz
+      steps:
+        - setup-ci: git clone https://gist.github.com/3d2388b2a7ba658cdcdaffa8cd874e50.git ci
+        - install: bundle install
+        - build: bundle exec jekyll build --source docs --destination _site
+        - package: tar -C _site -cvzf $RELEASE_FILE .
+        - tag: ./ci/git-tag.sh
+        - publish: ./ci/git-release.sh
```

Comment:
Need secrets to do git-tag and git-release:
https://github.com/screwdriver-cd/gitversion/blob/master/screwdriver.yaml#L27

Reply:
Added the secrets to the pipeline.